websites/R/R-a-d.io/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		//The client ID of the Application created at https://discordapp.com/developers/applications
+		clientId: "1073136952905318401",
+	}),
+	// Get current player status
+	playStatus = (): string => {
+		let status: string;
+		const playButtonText: string =
+			document.querySelector("#stream-play").textContent;
+		if (playButtonText === "Play Stream") status = "Stopped";
+		else if (playButtonText === "Stop Stream") status = "Playing";
+		else status = playButtonText;
+		return status;
+	},
+	// Get current song playing
+	currentSong = (): string => {
+		return document.querySelector("#np").textContent;
+	};
+
+presence.on("UpdateData", async () => {
+	// Current path
+	const path = window.location.pathname,
+		presenceData: PresenceData = {
+			// R/a/dio logo
+			largeImageKey: "logo",
+		};
+	if (path.includes("/news")) {
+		// On News page
+		presenceData.details = "Reading news...";
+		const titleElement = document.querySelector(
+			"#radio-container > section > div.container.main > div > div.panel.panel-default > div.panel-heading > h4 > a"
```

Comment:
Simplify this selector

Reply:
I'm not really sure if this can be simplified anymore, I got this from "Copy selector" in my browser devtools. Care to elaborate?

Reply:
Basically in this case you could look at the parent element's class:
![image](https://user-images.githubusercontent.com/42322979/218259192-47792927-4a74-4187-a44a-c9158c54eab3.png)

and then make a selector:
```
document.querySelector('[class="panel-title"] > a')
```

Reply:
Aah, I see. I forgot that `querySelector` can do this. My bad.

Reply:
> 

its fine lol